.github/workflows/continuous-delivery.yml

```diff
@@ -0,0 +1,76 @@
+name: Continuous Delivery
+
+on:
+    push:
+        tags:
+            - "v*"
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Setup MinGW
+              run: sudo apt-get install mingw-w64 -y
+
+            - name: Compile sbn64 for Windows (x86)
+              run: |
+                  i686-w64-mingw32-gcc -std=c99 -Wall -Wextra -O2 sbn64.c -o sbn64_Windows_x86.exe
```

Comment:
`CC` should be exported as an environment variable and `make` should be called, instead of manually building the binary by directly running the compiler.